FEATURES.md

```diff
@@ -0,0 +1,87 @@
+
+<u>__Writing the code__</u>
+
+Then the fun part of implementing the feature happens and that happens the way it typically does in an open source project. Once everything is done and assuming that the code is merged into the repo, the original feature proposal issue is closed out.
+
+The goal for this process is to have a single point of entry for all comers wanting to add a feature to the project. I hope this churns up a long pondered OpenSearch feature and that you write up your first feature proposal today!
+
+
+<u>__High level flow:__</u>
+| Step | Action | Who | Details |
+|------|--------|-----|---------|
+|1     | New proposal issue | Contributor / Feature proposal owner | Add label: [feature-proposal] |
+|2     | Conversation on issue |Maintainer, Contributor, Community| ✅ Feature? ✅ Scope? ✅ Distinct? ✅ Conflict? ✅ Atomic? |
+|3     | Approval of feature proposal | Maintainer |
+|4     | Create new META issue(s) and link the sub-tasks involved in delivering this feature | Maintainer, Contributor, Feature proposal owner | ✅ The proposal issue and design PR are linked to the parent META / child Meta issue., ✅ Child Meta issue that tracks all corresponding to targeted OpenSearch release version is created, ✅ The child META issue and the corresponding tasks are labelled with targeted OpenSearch version |
+|5     | Add meta issue(s) card specific to a release version to public roadmap | Maintainer | ✅ Add child Meta issue card to estimated release version |
+|6    | Pull request of feature code | Maintainer, Contributor, Feature proposal owner | ✅ Update the GitHub issues associated with a Meta issue on regular basis ✅ Move the issues to respective release as needed when working through the issues |
+|7    | Code review of feature code | Maintainer, Contributor, Feature proposal owner, Community
+|8    | Approval of code, Maintainer | ✅ Work with the maintainer to get the approval to merge the code |
+|9    | Merging of feature code | Maintainer | ✅ Maintainer will merge the code and add appropriate backport labels |
+|10    | Repeat step 10 to 13 |  Maintainer, Contributor, Feature proposal owner, Community | ✅ Repeat the steps until all Meta issues / Child META issues are closed |
+|11    | Close the proposal issue | Maintainer | ✅ Close the proposal once all the Meta issues are closed |
+
+# Appendix:
+
+## Meta issue format:
```

Comment:
This disagrees with the current template. Update https://github.com/opensearch-project/.github/blob/main/.github/ISSUE_TEMPLATE/FEATURE_REQUEST_TEMPLATE.md or create a new FEATURE_PROPOSAL_TEMPLATE.md and link to it. Make sure it's in the same format as the existing ones.